superset/connectors/sqla/models.py

```diff
@@ -47,8 +47,6 @@ def query(self, query_obj):
         error_message = None
         qry = db.session.query(Annotation)
         qry = qry.filter(Annotation.layer_id == query_obj['filter'][0]['val'])
-        qry = qry.filter(Annotation.start_dttm >= query_obj['from_dttm'])
-        qry = qry.filter(Annotation.end_dttm <= query_obj['to_dttm'])
```

Comment:
@hughhhh, why do you need to remove this? I'm worried it might bring too many annotations to the browser if we remove this — it would bring all holidays, instead of only the ones being displayed, eg.

Reply:
how about:
```
if query_obj['from_dttm']:
    qry = qry.filter(Annotation.start_dttm >= query_obj['from_dttm'])
```

Reply:
looking at the `annotations_json` endpoint we don't set the `from_dttm` or `to_dttm` at all. I just wanted to fix this endpoint to actually work because on master its not working at all.

https://github.com/apache/incubator-superset/blob/master/superset/views/core.py#L1118